### DISCUSSION TITLE
How to force pnpm to use remote packages instead of local packages in the workspace in production

### DISCUSSION THREAD

**Discussion by aladinflux** [pnpm/pnpm#4816] [Q&A]
So we have a mono repo with 3 nextjs apps and all modules in packages folder are published to npm registry. 
We want to give an option to pnpm so that in the production environment, it ignores the packages in the workspace and uses the npm registry.

This is to speed up the deployment in production, we use Vercel, we don't want all packages to be rebuilt during deployment.

**Comment by zkochan** [comment]
Maybe you need a deploy command?

Related issue: https://github.com/pnpm/pnpm/issues/4378

**Comment by aladinflux** [comment]
Well, currently the solution I found is a script that simple deletes the entry of those packages I don't want to be linked from `pnpm-workspace.yaml`, then it runs `pnpm i`. It works, but it feels quirky.
